scripts/codemod/helpers/codemodHelpers.js

```diff
@@ -0,0 +1,284 @@
+
+type GetImportsType = {
+  src: GenericType,
+  j: GenericType,
+};
+
+/**
+ * getImports: Returns an array of the import declaration in a file
+ */
+const getImports = ({ src, j }: GetImportsType): GenericType => src.find(j.ImportDeclaration);
+
+type GetJSXType = {
+  src: GenericType,
+  j: GenericType,
+};
+
+/**
+ * getJSX: Returns an array of the JSX elements in a file
+ */
+const getJSX = ({ src, j }: GetJSXType): GenericType => src.find(j.JSXElement);
+
+type GetLocalImportedNameType = {
+  importDeclaration: GenericType,
+  importedName: string,
+};
+
+/**
+ * getLocalImportedName: Returns the local named import node if it matches a name value
+ * E.g. import { Box } from 'gestalt & (importedName = Box) // Box
+ * E.g. import { Box as RenamedBox } from 'gestalt & (importedName = Box) // RenamedBox
+ */
+const getLocalImportedName = ({
+  importDeclaration,
+  importedName,
+}: GetLocalImportedNameType): GenericType =>
+  importDeclaration.specifiers
+    .filter((node) => node.imported.name === importedName)
+    .map((node) => node.local.name)[0];
+
+type ReplaceImportedNamedType = {
+  j: GenericType,
+  importDeclaration: GenericType,
+  previousCmpName: string,
+  nextCmpName: string,
+};
+
+/**
+ * replaceImportedNamed: Replaces the name of a named import
+ * E.g. previousCmpName = Box & nextCmpName =  RenamedBox
+ * imput: import { Box } from 'gestalt' >> output: import { RenamedBox } from 'gestalt'
+ */
+const replaceImportedName = ({
+  j,
+  importDeclaration,
+  previousCmpName,
+  nextCmpName,
+}: ReplaceImportedNamedType): GenericType =>
+  importDeclaration.specifiers.map((node) =>
+    node.imported.name === previousCmpName ? j.importSpecifier(j.identifier(nextCmpName)) : node,
+  );
+
+type SortImportedNamesType = { importSpecifiers: GenericType };
+
+/**
+ * sortImportedNames: Returns a sorted list of named imports
+ * E.g. imput: import { Pog, Box } from 'gestalt' >> output: import { Box, Pog } from 'gestalt'
+ */
+const sortImportedNames = ({ importSpecifiers }: SortImportedNamesType): GenericType =>
+  importSpecifiers.sort((a, b) => a.imported.name.localeCompare(b.imported.name));
+
+type SourceHasChangesType = { src: GenericType };
+
+/**
+ * sourceHasChanges: Adds a 'modified: true' key-value to the src object that indicates the file contains changes that  must be saved
+ */
+const sourceHasChanges = ({ src }: SourceHasChangesType): void => {
+  // eslint-disable-next-line no-param-reassign
+  src.modified = true;
+};
+
+type ReplaceImportNodePathType = {
+  j: GenericType,
+  nodePath: GenericType,
+  importSpecifiers: GenericType,
+  importPath: string,
+};
+
+/**
+ * replaceImportNodePath: Replaces an import declaration node with an updated one
+ * E.g. previousCmpName = Box & nextCmpName =  RenamedBox
+ * imput: <Box /> >> output: <RenamedBox />
+ */
+
+const replaceImportNodePath = ({ nodePath, importSpecifiers }: ReplaceImportNodePathType): void => {
+  // eslint-disable-next-line no-param-reassign
```

Comment:
@AlbertCarreras let's not disable this lint rule anywhere.

Reply:
I reduced the amount, leaving one.

Reply:
@christianvuerings Remaining usage is in codemodHelpers:155

Reply:
TODO follow up